hail/src/main/scala/is/hail/annotations/aggregators/RegionValuePearsonCorrelationAggregator.scala

```diff
@@ -0,0 +1,34 @@
+package is.hail.annotations.aggregators
+
+import is.hail.annotations.{Region, RegionValueBuilder}
+import is.hail.stats.PearsonCorrelationCombiner
+
+class RegionValuePearsonCorrelationAggregator extends RegionValueAggregator {
+  var combiner = new PearsonCorrelationCombiner()
+
+  def seqOp(region: Region, x: Double, xm: Boolean, y: Double, ym: Boolean) {
+    if (!xm && !xm)
```

Comment:
This should be `!xm && !ym`

Reply:
thanks! Will make sure this is tested too

Reply:
fixed. verified that my test caught this.